website/docs/cloud-docs/integrations/aws-service-catalog/index.mdx

```diff
@@ -0,0 +1,23 @@
+---
+page_title: Terraform Cloud for AWS Service Catalog
+description: >-
+  Provision infrastructure with HashiCorp Terraform Cloud inside AWS Service Catalog
+---
+
+# Terraform Cloud for AWS Service Catalog overview
+
+This integration allows administrators to curate a portfolio of pre-approved Terraform configurations on AWS Service Catalog. This enables end users like engineers, database administrators, and data scientists to deploy these Terraform configurations with a single action from the AWS interface. By combining Terraform Cloud with AWS Service Catalog, we’re combining a self-service interface that many customers are familiar with, AWS Service Catalog, with the existing workflows and policy guardrails of Terraform Cloud.
+
+<!-- BEGIN: TFC:only name:pnp-callout -->
+@include 'tfc-package-callouts/aws-service-catalog.mdx'
+<!-- END: TFC:only name:pnp-callout -->
+
+## Installation & Configuration
+
+To start using this integration, you'll need to install the [AWS Service Catalog Engine for Terraform Cloud](https://github.com/hashicorp/aws-service-catalog-engine-for-tfc) provided by HashiCorp on GitHub by following the [setup instructions](https://github.com/hashicorp/aws-service-catalog-engine-for-tfc#getting-started) provided in the README. If you run into any setup troubles along the way, the README also includes [troubleshooting steps](https://github.com/hashicorp/aws-service-catalog-engine-for-tfc#troubleshooting) that should help resolve common issues that you may encounter.
+
+With the engine installed, the necessary code and infrastructure to integrate the Terraform Cloud engine with AWS Service Catalog will automatically be configured. The setup can be completed in just a few minutes, and it only needs to be done once. Once the setup is complete, you can immediately start using AWS Service Catalog to develop and manage AWS Service Catalog products, and make them accessible to your end users across all your accounts.
```

Comment:
Similar suggestion as above: Should we generalize this a bit?:
```suggestion
With the engine installed, the necessary code and infrastructure to integrate the Terraform engine of your choice with AWS Service Catalog will automatically be configured. The setup can be completed in just a few minutes, and it only needs to be done once. Once the setup is complete, you can immediately start using AWS Service Catalog to develop and manage AWS Service Catalog products, and make them accessible to your end users across all your accounts.
```

Reply:
https://github.com/hashicorp/terraform-docs-common/pull/507#discussion_r1414335429